learning/web/content/modules/antipsychotics/CON155606_4.mdx

```diff
@@ -33,7 +34,7 @@ import Expander from '../../../src/components/Expander'
 * Antipsychotics should be used with caution and smaller doses selected in marked renal impairment to prevent cerebral toxicity. In renal insufficiency, dose reduction may be necessary especially if the drug is excreted by the kidney (eg amisulpride and sulpiride).  
   
 
-* Use an antipsychotic judiciously in the elderly, with careful and individual evaluation of likely benefits and risks-the risk of adverse events is greater in frail, elderly people, especially in the presence of other disorders (eg markedly impaired renal function and immobility)  
+* Use an antipsychotic judiciously in the elderly, with careful and individual evaluation of likely benefits and risks - the risk of adverse events is greater in frail, elderly people, especially in the presence of other disorders (eg markedly impaired renal function and immobility)  
```

Comment:
i think the dash is supposed to remain here because thats what the old website shows http://www.mhra.gov.uk/antipsychotics-learning-module/CON155606?useSecondary=&showpage=4